bitnami/jaeger/templates/agent/service.yml

```diff
@@ -0,0 +1,88 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name:  {{ include "jaeger.agent.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
```

Comment:
Use this instead. Applicable to all objects

```
  namespace: {{ include "common.names.namespace" . | quote }}
```

Reply:
Done [0769a67](https://github.com/bitnami/charts/pull/13480/commits/0769a677c4c2e08707747ef2c2e318be4572c462), [0769a67](https://github.com/bitnami/charts/pull/13480/commits/0769a677c4c2e08707747ef2c2e318be4572c462)